semantics/language/translation/decl/tagged.k

```diff
@@ -27,7 +27,8 @@ module C-DECL-TAGGED
           => addTag(X, makeStructFieldInfo(L))
           requires L =/=K .List
           [structural]
-     rule (.K => UNDEF("CDT1", "structs cannot be empty.", "6.7.2.1") )
+    //Undefinedness occurs when there are no named members in a struct. An empty struct is a contraint Violation. 
+     rule (.K => CV("CDT1", "structs cannot be empty.", "6.7.2.1") )
```

Comment:
This does seem correct although it's rather more complex than the current semantics supports, hence the issue Manasvi created. In any event, we will have to fix this part of the semantics at some point in the future.